cli/mender-artifact/main.go

```diff
@@ -114,6 +121,24 @@ func getCliContext() *cli.App {
 		Usage: "The group(s) the artifact depends on",
 	}
 
+	// Common Software Version flags
+	softwareVersionNoDefault := cli.BoolFlag{
+		Name:  noDefaultSoftwareVersionFlag,
+		Usage: "Disable the software version field for compatibility with old clients",
+	}
+	softwareVersionName := cli.StringFlag{
+		Name:  softwareNameFlag,
+		Usage: "Name of the key to store the software version: rootfs-image.NAME.version, instead of rootfs-image.version",
```

Comment:
This help string is incorrect for `write module-image`. The default there is `rootfs-image.PAYLOAD_TYPE.version`.

Reply:
Oh, sorry, I missed it.